test/account_multisig.test.js

```diff
@@ -0,0 +1,126 @@
+/* global BigInt */
+const nearApi = require('../lib/index');
+const fs = require('fs');
+const BN = require('bn.js');
+const testUtils  = require('./test-utils');
+const semver = require('semver');
+
+let nearjs;
+let startFromVersion;
+
+const {
+    KeyPair,
+    transactions: { functionCall },
+    InMemorySigner,
+    keyStores: { InMemoryKeyStore },
+    multisig: { AccountMultisig, MULTISIG_GAS, MULTISIG_DEPOSIT },
+    utils: { format: { parseNearAmount } }
+} = nearApi;
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 50000;
+
+const AccountMultisigInstance = async (account, keyMapping = ({ public_key: publicKey }) => ({ publicKey, kind: 'phone' })) => {
+    // modifiers to functions replaces contract helper (CH)
+    const { accountId } = account;
+    const keys = await account.getAccessKeys();
+    const accountMultisig = new AccountMultisig(nearjs.connection, accountId, {
+        // skip this (not using CH)
+        getCode: () => {},
+        sendCode: () => {},
+        // auto accept "code"
+        verifyCode: () => ({ success: true, res: '' }),
+        onResult: async () => {
+            const { requestId } = accountMultisig.getRequest();
+            console.log('requestId', requestId);
+            // set confirmKey as signer
+            const originalSigner = nearjs.connection.signer;
+            const tempKeyStore = new InMemoryKeyStore();
+            await tempKeyStore.setKey(nearjs.connection.networkId, accountId, accountMultisig.confirmKey);
+            nearjs.connection.signer = new InMemorySigner(tempKeyStore);
```

Comment:
seems like this pattern is repeating enough times that makes sense to extract as utility. `InMemorySigner.fromKeyPair` maybe?

Reply:
moved to separate issue